main.py

```diff
@@ -1 +1,208 @@
+                flower_count += 1
+            elif garden[rows][column] == "W":
+                weed_count += 1
+    return flower_count, weed_count
+
+
+# Ensures fulfillment of condittions with eachh passing day in the simulation where each flower and weed takes over adjacent squares
+
+
+def day_increment(garden, horizontal, vertical):
+    """Returns a version of the garden where a day has passed"""
+    for row in range(horizontal):
+        for col in range(vertical):
+            if garden[row][col] == "F":
+                item = "F"
+                garden = spread_up(garden, row, col, item)
+                garden = spread_down(garden, row, col, item)
+                garden = spread_right(garden, row, col, item)
+                garden = spread_left(garden, row, col, item)
+            elif garden[row][col] == "W":
+                item = "W"
+                garden = spread_up(garden, row, col, item)
+                garden = spread_down(garden, row, col, item)
+                garden = spread_right(garden, row, col, item)
+                garden = spread_left(garden, row, col, item)
+
+    return garden
+
+
+def grow_garden(garden, horizontal, vertical):
+    days = int(input("How many days do you want to simulate? "))
+
+    for _ in range(days):
+        garden = day_increment(
+            garden, horizontal, vertical
+        )  # Update garden with the latest state
+        print()
+        print("----------")
+        print()
+        display_garden(garden, horizontal, vertical)
+
+
+def main():
+
+    horizontal = int(input("Enter the number of rows in the garden: "))
+    vertical = int(input("Enter the number of columns in the garden: "))
+
+    # call make and display garden functions
+    garden = create_garden(horizontal, vertical)
+    display_garden(garden, horizontal, vertical)
+    cont = True
+    while cont:
+        # main code
+
+        # ask the user choice menu
+        choice = int(
+            input(
+                "What do you want to do? Plant,remove weeds, or count the number of plants and weeds. Enter 1, 2, 3: press 4 for passing N days "
+            )
+        )
+
+        if choice == 1:
+            x = int(input("Enter the x coordinate for where you want to plant: ")) - 1
+            y = int(input("Enter the y coordinate for where you want to plant: ")) - 1
+            if garden[x][y] != ".":
+                print("Invalid, item already present in that area.")
+            else:
+                garden[x][y] = "F"
+            display_garden(garden, horizontal, vertical)
+
+        elif choice == 2:
+            x = (
+                int(input("enter the x coordinate form where you want to remove weed "))
+                - 1
+            )
+            y = (
+                int(
+                    input(
+                        "Enter the ycoordinate form where you want ot remove weed :  "
+                    )
+                )
+                - 1
+            )
```

Comment:
You can convert this into a function to remove duplication for planting and removing. For example:
```python
def input_coords():
    # your code here
    return x, y
```